src/accel/TrackingAction.cc

```diff
@@ -0,0 +1,39 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file accel/TrackingAction.cc
+//---------------------------------------------------------------------------//
+#include "TrackingAction.hh"
+
+#include <G4Track.hh>
+
+#include "celeritas/phys/PDGNumber.hh"
+#include "celeritas/phys/ParticleParams.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct with Celeritas shared data.
+ */
+TrackingAction::TrackingAction() {}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Initialize Celeritas.
+ */
+void TrackingAction::PreUserTrackingAction(const G4Track* track)
+{
+    // Check against list of supported celeritas particles
+    PDGNumber pdg{track->GetDefinition()->GetPDGEncoding()};
+    if (true)
```

Comment:
Indeed, this is the first skeleton, but it may be better to add even a dummy interface to trigger a kind of offloading action.  Otherwise, we never collect a particle and never call the GPU transporter in this skeleton.  Ignore this comment if it is really intended as the zeroth pass of Geant4 integration.

Reply:
Yep, it really is a zeroth pass. I just wanted to get this to the point where we could start partitioning the work among multiple pull requests with people working in parallel.